torch/_inductor/lowering.py

```diff
@@ -6223,6 +6224,93 @@ def while_loop(cond_fn, body_fn, carried_inputs, additional_inputs):
     return list(map(TensorBox.create, result))
 
 
+@register_lowering(scan_op)
+def scan(combine_subgraph, init, inputs, dim, reverse, additional_inputs):
+    from torch._higher_order_ops.scan import _extract_carry_and_out, stack_y
+
+    num_init_leaves = len(init)
+    if any(map(is_triton, [init, inputs, *additional_inputs])):
```

Comment:
What's the purpose of this? We shouldn't need to disable cudagraphs here.

Reply:
I'm just copying over the cond and while_loop implementation. Let me remove it.